### ISSUE TITLE
Can't use nodeSelector with helm chart

### ISSUE THREAD

**Issue by kuzaxak** [aws/aws-node-termination-handler#202]
```
helm upgrade --install aws-node-termination-handler \
  --namespace kube-system \
  --set nodeSelector.lifecycle=spot \
  eks/aws-node-termination-handler
```

Gives you an error: 
```
Error: template: aws-node-termination-handler/templates/daemonset.linux.yaml:157:26: executing "aws-node-termination-handler/templates/daemonset.linux.yaml" at <8>: wrong type for value; expected string; got map[string]interface {}
```

Version:
```
helm3 version
version.BuildInfo{Version:"v3.2.4", GitCommit:"0ad800ef43d3b826f31a5ad8dfbb4fe05d143688", GitTreeState:"clean", GoVersion:"go1.13.12"}
```

**Comment by haugenj** [comment]
Thanks for bringing this up!

We released chart version 1.6 last night that includes support for Windows, and after a quick look it seems we might not be passing the value for nodeSelector correctly. We'll investigate further as soon as we can

**Comment by bwagner5** [comment]
released fix in eks-charts
